### ISSUE TITLE
Fix publisher unhautorized exception

### ISSUE THREAD

**Issue by SekoduNord** [CSumm/assignment4#29]
I updated the code this morning with a new stub according to the new getPublisherInfo() method.
We now received an unhautorized user exception error.
I try different approaches to solve this issue, but nothing worked. We will need somehow to set the authorization field in the header of the stub located in the PublisherClient class.

**Comment by CSumm** [comment]
Has this issue been resolved? Who is this assigned to?

**Comment by SekoduNord** [comment]
Nope, I tried again yesterday with the same result... I also try to put back the call from ClientCaller wthout any success. Any help would be appreciated.

**Comment by CSumm** [comment]
Graeme managed to solve the authorization issue. If you want Graeme to give him the green light to push, let us know

**Comment by SekoduNord** [comment]
For sure. Let me know when you're done. :-)
